### DISCUSSION TITLE
Unsafe code sections

### DISCUSSION THREAD

**Discussion by hnisula** [hsutter/cppfront#1160] [Q&A] [answered]
I have not been able to find any more information about unsafe code other than the goal of having some kind of support for it. Is this so far completely unsupported or is it required to use mixed files to use unsafe code with cppfront?

**Comment by hsutter** [comment, marked as answer]
Thanks for the question!

I intend to implement something like `[[suppress]]` to suppress things like unwanted bounds checks, but I haven't done that yet.

I don't intend to allow all Cpp1 unsafe constructs... you're correct, you can always write Cpp1 for that.
